EventFilter/GEMRawToDigi/plugins/GEMDigiToRawModule.cc

```diff
@@ -98,7 +98,7 @@ void GEMDigiToRawModule::produce(edm::StreamID iID, edm::Event& iEvent, edm::Eve
   auto gemROMap = runCache(iEvent.getRun().index());
 
   std::vector<std::unique_ptr<AMC13Event>> amc13Events;
-  amc13Events.reserve(FEDNumbering::MAXGEMFEDID - FEDNumbering::MINGEMFEDID + 1);
+  amc13Events.reserve(FEDNumbering::MAXME0FEDID - FEDNumbering::MINGEMFEDID + 1);
```

Comment:
`FEDNumbering::MAXGEMFEDID` (as well as the ME0 flavour) is only used here and in GEMRawToDigiModule.cc: `MINME0FEDID` is never used (i.e. you always loop from `MINGEMFEDID` to `MAXGEMFEDID`)
Perhaps it would be worth considering defining only
```
    MINGEMFEDID = 1467, 
    MAXGEMFEDID = 1478,
```
for all GEM's, including ME0: what do you think (preparing to eventually get rid of ME0)?

Reply:
I was planning to do this later, but since ME0 fed id's are not used, I've added this too.